### DISCUSSION TITLE
Default user password

### DISCUSSION THREAD

**Discussion by shahvand** [librenms/docker#418] [General]
Hi im setup with docker compose and traefik
and run on the web im dont have user and passowrd 
cant find on the environment

**Comment by murrant** [comment]
The first time you access the web, you set up a username and password.  Otherwise you will have to connect to the cli and create a user by running `docker compose exec --user librenms librenms lnms user:add --role=admin`  (adjust to match your environment if needed)

**Comment by shahvand** [comment]
Not enough arguments (missing: "username").

**Comment by shahvand** [comment]
this command corect
docker compose exec --user librenms librenms lnms user:add [username] --role=admin
